.github/workflows/pr_push.yml

```diff
@@ -112,6 +112,9 @@ jobs:
     - name: Check cmake-format
       run: cmake --build build --target cmake-format-check
 
+    - name: Check formatting with black
```

Comment:
nit: I think these steps' names could refer to the language of files formatted, like: C/C++ formatting check etc.

Reply:
voila